src/canvas/canvasRenderer.js

```diff
@@ -0,0 +1,166 @@
+//////////////////////////////////////////////////////////////////////////////
+/**
+ * Create a new instance of class canvasRenderer
+ *
+ * @class
+ * @extends geo.renderer
+ * @param canvas
+ * @returns {geo.gl.canvasRenderer}
```

Comment:
Is this  a typo: @returns {geo.gl.canvasRenderer} should be
@returns {geo.canvasRenderer}